node/utils/createOrdersToReturnSummary.ts

```diff
@@ -37,7 +37,7 @@ export const createOrdersToReturnSummary = async (
     { page: 1, pageSize: 100 },
     ['items', 'refundData', 'refundPaymentData'],
     undefined,
-    `orderId=${orderId}`
+    `orderId=${orderId} AND status <> cancelled`
```

Comment:
I'm just ignoring any request with their status as cancelled, so the whole resolver won't take them into consideration

Reply:
That is a nice trick. I didn't know about that. Super!